src/View/User/UserMapView.fxml

```diff
@@ -25,6 +25,7 @@
 <?import javafx.scene.control.TableView?>
 <?import javafx.scene.control.TableColumn?>
 
+<?import javafx.scene.shape.Polygon?>
```

Comment:
Also can you make the arrows just a tad transparent and spaced a little more from the combo box